react app/src/App.css

```
@@ -1,42 +1,138 @@
+  margin-top: 1rem;
 }
 
-@keyframes logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+button,
+.fileInput {
```

Stylelint (17.2.0)

[error] Expected class selector ".fileInput" to be kebab-case (selector-class-pattern)

(selector-class-pattern)